spring-test/src/test/java/org/springframework/test/web/servlet/htmlunit/HtmlUnitRequestBuilderTests.java

```diff
@@ -178,7 +178,7 @@ public void buildRequestContextPathUsesFirstSegmentByDefault() {
 
 	@Test
 	public void buildRequestContextPathUsesNoFirstSegmentWithDefault() throws MalformedURLException {
-		webRequest.setUrl(new URL("http://example.com/"));
+		webRequest.setUrl(new URL("https://example.com/"));
```

Comment:
I thought we shouldn't be using `example.com` anymore?

Reply:
Please see See https://github.com/spring-io/nohttp/tree/master/nohttp#thought-process